GoogleForCreators/web-stories-wp
# Stories block: Fix story selector screen styling issue

## Context



## Summary

Story selector screen styling issue. This happened when design system change, making the CSS wrong. 

## Relevant Technical Choices

Adding a E2E and screenshot, should mean this doesn't happen again. 

## To-do



## User-facing changes

<img width="761" alt="Screenshot 2021-06-25 at 14 06 26" src="https://user-images.githubusercontent.com/237508/123429209-98439300-d5be-11eb-9176-119ea53c7dd6.png">
## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Create a new post.
2. Create a web stories block. 
3. Select, selected stories. 
4. Select a layout type ( type is not important ). 
5. Click selected stories button. 
6. See select stories model. 

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7526
